packages/mobile/src/escrow/saga.test.ts

```diff
@@ -12,9 +12,14 @@ import {
 } from 'src/escrow/actions'
 import { reclaimFromEscrow, transferToEscrow } from 'src/escrow/saga'
 import { newTransactionContext } from 'src/transactions/types'
-import { Currency } from 'src/utils/currencies'
 import { getConnectedAccount, unlockAccount, UnlockResult } from 'src/web3/saga'
-import { mockAccount, mockE164Number, mockE164NumberHash, mockE164NumberPepper } from 'test/values'
+import {
+  mockAccount,
+  mockCusdAddress,
+  mockE164Number,
+  mockE164NumberHash,
+  mockE164NumberPepper,
+} from 'test/values'
 
 describe(transferToEscrow, () => {
```

Comment:
Looks like we're missing a happy path test for this feature 😢

Reply:
Added!